### DISCUSSION TITLE
How about pass throught devices?

### DISCUSSION THREAD

**Discussion by McKpol** [xoureldeen/Vectras-VM-Android#50] [Q&A]
Is there a possibility to pass throught Bluetooth or USB, something like pendrive or HID device?

**Comment by xoureldeen** [comment]
Thats qemu thing

qemu.org

**Reply by McKpol** [reply]
I mean yeah i know, but might android block it or something.
soo I'm asking.

**Reply by xoureldeen** [reply]
I still didn't tested it, u need to try it in th terminal in Vectras VM with root permission
